### PR TITLE
Version number displayed on the navbar, moved github link to the footer

### PR DESCRIPTION
**Pull request by 4ydan**
#510 

## Basics

- [x] I added a line to [/doc/CHANGELOG.md](/doc/CHANGELOG.md)
- [x] The PR is rebased with current master.
- [x] Details of what you changed are in commit messages.
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [x] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added unit tests for my code
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

**Comment by 4ydan** [comment]
Is this ok? On the top left
image.
and Github on the footer with a logo 
image
I didnt do any additional API call, I just used the existing one to write the version to the cache.
Should I check the backend version against the frontend version, throw any exception or anything special?

**Comment by 4ydan** [comment]
Is there a reason we are doing multiple config api calls to the backend?
And also I will have to make this poll regularly the backend to find out if the versions dont match?
image

**Comment by markus2330** [comment]
> I will have to make this poll regularly the backend to find out if the versions dont match?

No, you do not need to poll, see #578

File: frontend/src/components/Layout/Navbar.tsx [hunk 1]
```diff
@@ -16,7 +19,12 @@ const Navbar = () => {
   const darkMode = useDarkModeStore((state) => state.darkMode);
   const { t } = useTranslation(['routes']);
   const auth = useSafeAuth();
-
+  const backendVersion = sessionStorage.getItem('version');
+  useEffect(() => {
+    if (backendVersion != 'frontendVersion') {
+      toast(`Ihre Version ist nicht mehr aktuell. Bitte laden Sie die Seite neu.`);
```
Comment (absurd-turtle) [inline frontend/src/components/Layout/Navbar.tsx:25]: You can translate the string using useTranslation

File: frontend/src/components/Layout/Footer.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import ButtonLink from '../Button/ButtonLink';
 import { useTranslation } from 'react-i18next';
+import { FaGithub } from 'react-icons/fa';
```
Comment (absurd-turtle) [inline frontend/src/components/Layout/Footer.tsx:3]: Is the react-icons dependency useful for anything else?
If not maybe just use an svg version of the github logo

File: frontend/src/components/Layout/Navbar.tsx [hunk 3]
```diff
@@ -16,7 +19,12 @@ const Navbar = () => {
   const darkMode = useDarkModeStore((state) => state.darkMode);
   const { t } = useTranslation(['routes']);
   const auth = useSafeAuth();
-
+  const backendVersion = sessionStorage.getItem('version');
+  useEffect(() => {
+    if (backendVersion != 'frontendVersion') {
```
Comment (absurd-turtle) [inline frontend/src/components/Layout/Navbar.tsx:24]: Can this actually happen?

**Review by absurd-turtle — CHANGES_REQUESTED** [review]

File: frontend/src/components/Layout/Navbar.tsx (hunk 3 above)
Comment (markus2330) [reply frontend/src/components/Layout/Navbar.tsx:24]: The idea was to check if the backend upgraded. Is this the wrong position to check for?

File: frontend/src/components/Layout/Footer.tsx (hunk 2 above)
Comment (4ydan) [reply frontend/src/components/Layout/Footer.tsx:3]: Thank you, good idea, I will do that.

File: frontend/src/components/Layout/Navbar.tsx (hunk 1 above)
Comment (4ydan) [reply frontend/src/components/Layout/Navbar.tsx:25]: Oh thats a typo by me, shouldnt be a string

File: frontend/src/components/Layout/Navbar.tsx (hunk 3 above)
Comment (markus2330) [reply frontend/src/components/Layout/Navbar.tsx:24]: Yes, remove it for now, as it is not the scope of this issue.

**Comment by markus2330** [comment]
Don't worry, this happens.

Actually you should be able to simply abort a git rebase with `--abort`.

Otherwise you could simply restore your branch using the old SHA-ID of the latest commit (of the branch as it was before). So probably [30c0a89](https://github.com/ElektraInitiative/PermaplanT/pull/574/commits/30c0a89340f6f71e82813461fd9af65bfd365b71)

What does `git status` say?

> Oh maybe I should have never done git rebase, right? Only rebase when it cant merge automatically?

Yes, rebase only if needed. And if rebase is difficult (which is the case if you changed a conflicting file many times) you can simply merge master (then you only need to resolve the conflicts once and not per commit). Merging master from PRs gives a more ugly git history but does the same job.

File: frontend/src/components/Layout/Footer.tsx [hunk 4]
```diff
@@ -11,6 +11,17 @@ export const Footer = () => {
         <span className="text-sm text-secondary-500 dark:text-secondary-300 sm:text-center">
           © 2023 PermaplanT
         </span>
+        <span className="text-sm text-secondary-500 dark:text-secondary-300 sm:text-center">
+          <a
+            href="https://github.com/ElektraInitiative/PermaplanT"
+            style={{ display: 'flex', alignItems: 'center' }}
+          >
+            <svg viewBox="0 0 1024 1024" fill="currentColor" height="1em" width="1em">
```
Comment (absurd-turtle) [inline frontend/src/components/Layout/Footer.tsx:19]: Storing the svg in a file and importing it makes the markup more clean and the svg reusable.
You can look at the icons we use in different places as a reference.

**Review by absurd-turtle — CHANGES_REQUESTED** [review]

**Comment by 4ydan** [comment]
jenkins build please

**Pull request merged**